### PR TITLE
Onclick feedback when the user clicks to copy the payment request

### PR DESCRIPTION
**Pull request by bumi**
@reznord how can this code be implemented nicer? :)

### CODE REVIEW

**Comment by reznord** [comment]
This is much cleaner and easier without a lot of noise as well.

**Review by reznord — APPROVED** [review]

File: src/components/icons/copy/index.js [hunk 1]
```diff
@@ -1,9 +1,8 @@
 import { h } from "preact";
 
-const Copy = () => (
-  <svg width="15" height="15" fill="none" viewBox="0 0 15 15">
+const Copy = ({active}) => (
+  <svg className={active ? "copy active" : "copy"} width="15" height="15" fill="none" viewBox="0 0 15 15">
```
Comment (bumi) [inline src/components/icons/copy/index.js:4]: is this the correct way to do this? @reznord

Comment (reznord) [reply src/components/icons/copy/index.js:4]: Yes, this is how we normally change the colors of the svg.

Comment (bumi) [reply src/components/icons/copy/index.js:4]: and regarding the JS? and passing in the state?

**Pull request merged**